akka-sample-spring-extension-java/src/main/java/org/example/akka/actor/MyActorService.java

```diff
@@ -0,0 +1,22 @@
+package org.example.akka.actor;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class MyActorService {
+
+	public MyActorService() {
+
+	}
+
+	@PostConstruct
+	public void init() {
+		System.out.println("Created service as bean");
+	}
+
+	public void printService(String string) {
+		System.out.println("Printed with bean: " + string);
+	}
+}
```

Comment:
Would a typical spring "Service" contain mutable state and be injected in multiple actors? In that case please add a comment here that the service must be thread safe.

Reply:
It's not really a state because it's not an attribute. The service just "processes" the input string, in this case by printing it to console. A very simple example to show that the Autowired annotation works.

Reply:
Yep, I got that the sample is stateless, was more thinking of how it would be in a real(tm) spring app. People tend to copy paste these sample and build whole applications so some friendly comment-nudging in the right direction is often good.

Reply:
Oh sure, I'll do that.